### PR TITLE
add Motech tab under a feature flag

### PR DESCRIPTION
**Pull request by dannyroberts**
as a workspace for development

Nothing much here yet, just boilerplate:

screen shot 2017-04-12 at 1 05 07 pm

### CODE REVIEW

File: corehq/apps/motech/views.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    @property
+    def page_context(self):
+        return {
+            'openmrs_instance_form': self.openmrs_instance_form,
+        }
+
+    def post(self, request, *args, **kwargs):
+        if self.openmrs_instance_form.is_valid():
+            # self.openmrs_instance_form.save(self.request.couch_user, self.domain)
+            messages.success(request, _("Your OpenMRS server settings have been saved!"))
```
Comment (dimagimon) [inline corehq/apps/motech/views.py]: F821 undefined name '_'

**Comment by dannyroberts** [comment]
@benrudolph @proteusvacuum

File: corehq/apps/motech/views.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    @property
+    def page_context(self):
+        return {
+            'openmrs_instance_form': self.openmrs_instance_form,
+        }
+
+    def post(self, request, *args, **kwargs):
+        if self.openmrs_instance_form.is_valid():
+            # self.openmrs_instance_form.save(self.request.couch_user, self.domain)
```
Comment (benrudolph) [inline corehq/apps/motech/views.py]: did you mean to leave this commented

**Review by benrudolph — APPROVED** [review]

File: corehq/apps/motech/views.py (hunk 2 above)
Comment (dannyroberts) [inline corehq/apps/motech/views.py]: oops, no I'll clean this up. I'm actively developing over this anyway, but no harm pushing a little cleanup in the meantime.

**Review by benrudolph — APPROVED** [review]

File: corehq/apps/motech/views.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+from corehq.apps.domain.views import BaseDomainView
+from corehq.apps.motech.forms import OpenmrsInstanceForm
+from corehq.toggles import MOTECH
+from dimagi.utils.decorators.memoized import memoized
+
+require_motech_permissions = lambda fn: MOTECH.required_decorator()(domain_admin_required(fn))
+
+
+class MotechSection(BaseDomainView):
+    section_name = ugettext_lazy("Motech")
```
Comment (proteusvacuum) [inline corehq/apps/motech/views.py]: lol, what is "Motech" in French? 
Môtech?

Comment (proteusvacuum) [inline corehq/apps/motech/views.py]: also, is "MOTECH" not an acronym? Should it be all caps?

**Pull request merged**